src/class04/comparation.js

```diff
@@ -0,0 +1,10 @@
+var x = 5
```

Comment:
Replace this excerpt with the following exercise:

"Write a program to read 3 integer values ​​(consider not
equal values ​​will be read) and write them in ascending order."